presto-hive-hadoop2/bin/common.sh

```diff
@@ -1,5 +1,3 @@
-#!/bin/bash
```

Comment:
I remember you wanting me to add this previously, so that editors and such would know this is a bash file (without relying on the extension).

Reply:
Right. Perhaps even Intellij was begging me for a shebang. In VIM world, one would simply add `#!bash`. 

Will revert.